### PR TITLE
Fixes an empty layout on Android during startup

### PR DESCRIPTION
**Pull request by mortoray**
fixes https://github.com/fusetools/fuselibs-public/issues/1096

This PR contains:
- [x] Changelog
- [ ] ~Documentation~
- [x] Tests - ManualTestApp will go all red if it detects one of the symptoms

### CODE REVIEW

File: Source/Fuse.Android/AndroidApp.uno [hunk 1]
```diff
@@ -53,7 +53,7 @@ namespace Fuse
 
 			Uno.Platform.Displays.MainDisplay.Tick += OnTick;
 		}
-
+
```
Comment (kusma) [inline Source/Fuse.Android/AndroidApp.uno]: This seems silly

**Comment by mortoray** [comment]
@kusma I thought about a hard error, but it makes the fix more challenging. We have several controls that end up reading frame state prior to the viewport being ready, things like `ClientPanel`, `StatusBarFrame`, or triggers like `WhileWindowPortrait`, and the `window()`. There's no [easy] way to defer their creation, nor do they gain anything from having "unknown" values -- they all still need actual values.

So while not perfect, this patch fixes the most costly issue: the layout pass before ready. All those other bits don't gain us anything in performance, and will actually make the code more complex, thus I don't think it's worth it.

**Comment by kusma** [comment]
Right. If so, I think the right thing is to not mark the issue as fixed, but reduce the severity of it after this PR.

**Comment by mortoray** [comment]
I've created a new issue instead https://github.com/fusetools/fuselibs-public/issues/1102  It better describes the remaining situation.

**Pull request merged**